.env.example

```diff
@@ -51,6 +51,25 @@ COSY_TTS_KEY=
 # ElevenLabs TTS key
 ELEVENLABS_TTS_KEY=
 
+# Extension: litellm
+# Parameter Configuration Methods
+# Method 1
+#    Using Environment Variables, refer to https://docs.litellm.ai/docs/providers
+#    For example:
+#        OpenAI
+#            OPENAI_API_KEY=<your-api-key>
+#            OPENAI_API_BASE=<openai-api-base>
+#        AWS Bedrock
+#            AWS_ACCESS_KEY_ID=<your-aws-access-key-id>
+#            AWS_SECRET_ACCESS_KEY=<your-aws-secret-access-key>
+#            AWS_REGION_NAME=<aws-region-name>
+# Method 2
+#    Passing Parameters, specify LITELLM_API_KEY and LITELLM_PROVIDER
+#    provider: refer to the provider_list in https://github.com/BerriAI/litellm/blob/main/litellm/__init__.py
+LITELLM_API_KEY=
+LITELLM_MODEL=gpt-4o-mini
+LITELLM_PROVIDER=
```

Comment:
DON'T DO THIS.

Reply:
Currently, two methods for using litellm are provided. If adjustments are needed, what is the desired outcome?